docs/openstack-deploy-cli.md

```diff
@@ -0,0 +1,47 @@
+# Openstack Deploying the Command Line Tools
+
+Before we can get started we need to install a few things.
+
+#### Installing Python
+
+Installed by default on Mac OS X.
+
+Many Linux distributions provide packages to make setuptools easy to install. Search your package manager for setuptools to find an installation package. If you cannot find one, download the setuptools package directly from https://pypi.python.org/pypi/setuptools.
```

Comment:
I think we should point folks at the official upstream docs for pip found here. https://pip.pypa.io/en/stable/installation

Reply:
Updated